### PR TITLE
ENG-870: add events order to list events filter

### PR DESCRIPTION
**Pull request by efiShtain**
Notes for reviewers:

- much easier to go commit by commit 
- I fixed a bug in events order + added tests to make this PR work

### CODE REVIEW

File: proto/autokitteh/events/v1/svc.proto [hunk 1]
```diff
@@ -40,6 +40,7 @@ message ListRequest {
   string connection_id = 2;
   string event_type = 3;
   uint32 max_results = 4;
+  string order = 5;
```
Comment (itayd) [inline proto/autokitteh/events/v1/svc.proto]: add a comment for values or make an enum

File: sdk/sdkservices/events.go [hunk 2]
```diff
@@ -14,8 +14,14 @@ type ListEventsFilter struct {
 	Limit             int
 	CreatedAfter      *time.Time
 	MinSequenceNumber uint64
+	Order             string
 }
 
+const (
+	ListOrderAscending  = "ASC"
+	ListOrderDescending = "DESC"
+)
```
Comment (itayd) [inline sdk/sdkservices/events.go]:
```suggestion
	Order             ListOrder
}

type ListOrder string

const (
	ListOrderAscending  ListOrder = "ASC"
	ListOrderDescending ListOrder = "DESC"
)
```

this way it's clear what the expected values are from the type

**Review by itayd — APPROVED** [review]

**Review by itayd — APPROVED** [review]

**Review by itayd — APPROVED** [review]
see comment

File: proto/autokitteh/events/v1/svc.proto (hunk 1 above)
Comment (efiShtain) [reply proto/autokitteh/events/v1/svc.proto]: added comment

File: sdk/sdkservices/events.go (hunk 2 above)
Comment (efiShtain) [reply sdk/sdkservices/events.go]: changed. nice one. I like.

**Pull request merged**